misago/conf/defaults.py

```diff
@@ -344,3 +344,8 @@
     'vi',
     'zh-cn', 'zh-hans', 'zh-tw',
 ]
+
+
+# Specifies the amout of time we can saves the user ips in days
+
+MISAGO_IP_STORE_TIME = 50
```

Comment:
This should be `MISAGO_IP_STORE_TIME = None`. We want old IP's anonymization disabled on existing forums unless admin enables it by editing their `settings.py`